stellar-multitenant/pkg/tenant/fixtures.go

```diff
@@ -175,3 +178,24 @@ func ApplyMigrationsForTenantFixture(t *testing.T, ctx context.Context, dbConnec
 	_, err = db.Migrate(dsn, migrate.Up, 0, authmigrations.FS, db.StellarPerTenantAuthMigrationsTableName)
 	require.NoError(t, err)
 }
+
+func PrepareDBForTenant(t *testing.T, dbt *dbtest.DB, tenantName string) string {
```

Comment:
Can you add the `t.Helper()` flag at the beginning of this function? So this function will be marked as a helper function. It will be skipped in some flows (printing file information, coverage, etc...)

Reply:
addressed in 09e7ad2bf5ceaff937ffe6cd59ce3796e4e93e11
Thank you!